src/game/SpellEffects.cpp

```diff
@@ -1149,6 +1149,13 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
 
                     return;
                 }
+                case 32027:                                 // Expedition Flare
+                {
+                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
```

Comment:
also, if you would care to note, every other dummy effect with a CastItem has a check for the CastItem, and all dummy effects that come from items send along their CastItem; so it is not something we want to avoid doing, we would much rather want to find out why our CastItem is getting nullified. Again, my guess would be because it's consumed on use.